README.md

```diff
@@ -68,6 +68,46 @@ The collection is also available as `ansible-collection-theforeman-foreman` from
 
 After installing the RPM, you can use the modules in the same way as when they are installed directly from Ansible Galaxy.
 
+## Installation From Source
+
+For development or testing purposes, you can install the collection from sources. For production usage, see the instructions above on installing the latest stable release.
```

Comment:
```suggestion
For development or testing purposes, you can install the collection from source git repository. For production usage, see the instructions above on installing the latest stable release.
```

Reply:
Addressed here, and in the commit message as well.